### PR TITLE
additional fixes for new label spans

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
plz merge asap due to failing tests

### CODE REVIEW

File: tests/browser/features/step_definitions/label_steps.rb [hunk 1]
```diff
@@ -78,13 +78,13 @@
   on(ItemPage) do |page|
     expect(page.first_heading?).to be true
     page.wait_for_label(value)
-    expect(page.first_heading_element.text).to eq value + '(' + @items[item_handle]['id'] + ')'
+    expect(page.entity_label_span).to eq value
+    expect(page.entity_id_span).to eq '(' + @items[item_handle]['id'] + ')'
```
Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/label_steps.rb]: Is there a reason you are not using string interpolation?

https://github.com/bbatsov/ruby-style-guide#string-interpolation

Comment (WMDE-Fisch) [inline tests/browser/features/step_definitions/label_steps.rb]: nope :p

Comment (Benestar) [inline tests/browser/features/step_definitions/label_steps.rb]: Personally I think using + is much more readable. Not sure what kind of style guidelines we have for ruby.

Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/label_steps.rb]: Well, the link I have provided is _the_ style guide we are using. ;)

Comment (Benestar) [inline tests/browser/features/step_definitions/label_steps.rb]: Then go ahead :-)

Comment (WMDE-Fisch) [inline tests/browser/features/step_definitions/label_steps.rb]: I agree with Benestar for most cases but I respect the given guidelines. ;-)

**Pull request merged**